lib/hijack/mongo-driver-events.js

```diff
@@ -74,8 +77,9 @@ Meteor.startup(() => {
 
   client = _client;
```

Comment:
The check on line 73 won't work with version 4 of the driver since it doesn't have a useUnifiedTopology option.